frontend/webapp/containers/main/actions/action-drawer/build-card.ts

```diff
@@ -19,19 +20,20 @@ const buildCard = (action: ActionDataParsed) => {
     },
   } = action;
 
-  const arr = [
+  const arr: DataCardRow[] = [
     { title: 'Type', value: type },
-    { title: 'Status', value: String(!disabled) },
-    { title: 'Monitors', value: signals.map((str) => str.toLowerCase()).join(', ') },
-    { title: 'Name', value: actionName || 'N/A' },
-    { title: 'Notes', value: notes || 'N/A' },
+    { title: 'Name', value: actionName },
+    { title: 'Notes', value: notes },
+    { type: 'divider' },
+    { title: 'Status', type: 'active-status', value: String(!disabled) },
```

Comment:
second look at this file made me think that this file is getting a bit complex with repeated logic and a long function body. it might be worth re struct it